internal/terraform/transform_orphan_resource.go

```diff
@@ -26,9 +26,17 @@ type OrphanResourceInstanceTransformer struct {
 	// Config is the root node in the configuration tree. We'll look up
 	// the appropriate note in this tree using the path in each node.
 	Config *configs.Config
+
+	// There are no orphans when doing a full destroy
+	destroyPlan bool
```

Comment:
Same thought here as about `ConfigTransformer` above: can we make this flag have a name that describes more specifically what effect it has?